src/pypromice/L0toL1.py

```diff
@@ -177,6 +191,206 @@ def _flagNAN(ds, flag_file=None):
         # flag_LUT.loc["NAN"]['value']
     return ds
 
+
+def _adjustData(ds, var_list=[], skip_var=[]):
```

Comment:
This currently does not discern whether to flag and adjust `tx`, `STM` or `raw` data. I can foresee instances where only one `L0` data type needs to be adjusted. Maybe we need a column in the flag `.csv` file identifying which data type (or all) to flag and adjust?

Reply:
That might be true. In the historical GC-Net data, transmissions are much more noisy than raw logger files.
But I haven't seen such a big difference in the Iridium transmissions. 
That is also why we need to have regularly updated data treatment reports. Because some adjustments or filters might become obsolete or inadequate when a logger file replaces the transmissions.
I'll open an issue about it and it can be fixed later?

Reply:
Sounds good.